apps/api/src/storage/adapters/sqlite.adapter.ts

```
@@ -95,11 +96,112 @@ import type {
+      );
+    }
+  } catch (err) {
+    const message = err instanceof Error ? err.message : String(err);
+    // "no such table" is the only benign case, and it cannot happen after
+    // createSchema. Anything else means the duplicate-pending index may not
+    // exist, and swallowing that silently ships the race this migration is
+    // here to close.
+    if (/no such table/i.test(message)) {
+      return;
+    }
+    throw new Error(`memory_proposals integrity migration failed: ${message}`);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)